code/src/Radio.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+export const Radio = ({ handleRadioInput }) => {
+  return <div className="question" tabIndex="0">
+      <fieldset>
+        <legend tabIndex="0">Which character has the best callsign?</legend>
+        <label for="mav" tabIndex="0">Maverick</label> 
+        <input type="radio" value="Maverick" id="mav" onChange={e => handleRadioInput(e.target.value)} required/>
+        <label for="goose" tabIndex="0">Goose</label> 
+        <input type="radio" value="Goose" id="goose" onChange={e => handleRadioInput(e.target.value)}/>
+        <label for="sundown" tabIndex="0">Sundown</label> 
+        <input type="radio" value="Sundown" id="sundown" onChange={e => handleRadioInput(e.target.value)}/>
```

Comment:
I can see that you added "required" in one the radio buttons, however I could go through the form without choosing any of them, I had the same issue and I solved it but also adding a "name"  and "checked" attributes 😊

Reply:
I saw your solution and that was really nice! I do not know exactly what I am doing wrong but I can not get the required to work